pkg/user/signer.go

```diff
@@ -153,6 +153,7 @@ func (s *Signer) SubmitPayForBlob(ctx context.Context, blobs []*blob.Blob, opts
 		return nil, err
 	}
 	if resp.Code != 0 {
+		// TODO(htienv): Should return nil here? 
```

Comment:
good question. It seems odd to return a response and an error from this function.

Reply:
Yes, I think need reconsider for ensure it doesn't affect other things

Reply:
I'm not strongly opinionated on this but do you mind removing your handle from the TODO comment?

```suggestion
		// TODO: should this return nil? 
```

or removing the line all together and creating a GH issue?

Reply:
Yes, thank you!